lib/engine.js

```diff
@@ -293,6 +293,7 @@ module.exports = function container (get, set, clear) {
               msg('order status: ' + order.status)
               if (api_order.status === 'done') {
                 order.time = new Date(api_order.done_at).getTime()
+                order.price = api_order.price || order.price // Use actual price if possibile. In market order the actual price (api_order.price) should be very different from trade price
```

Comment:
`In market order the actual price (api_order.price) should be very different from trade price` should be or could be?

Reply:
Could be*, and very often is :)